src/main/resources/com/google/api/codegen/csharp/gapic_csproj.snip

```diff
@@ -0,0 +1,62 @@
+# packageInfo: PackageInfoView
+@snippet generate(packageInfo)
+  <?xml version="1.0" encoding="utf-8"?>
+  <Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+
+      <!-- TODO: Version defaults to 1.0.0, edit as required -->
+      <Version>{@packageInfo.version}</Version>
+      
+      <!-- TODO: Nuget packaging option -->
+      <!--
+        <Description>
+          {@packageInfo.serviceTitle}
+          {@packageInfo.serviceDescription}
+        </Description>
+        <PackageTags>PubSub;Google;Cloud</PackageTags>
+        @if packageInfo.fileHeader.copyrightLines.size > 1
+          <Copyright>
+            @join line : packageInfo.fileHeader.copyrightLines
+              {@line}
+            @end
+          </Copyright>
+        @else
+          <Copyright>{@packageInfo.fileHeader.copyrightLines.get(0)}</Copyright>
+        @end
+        <Authors>{@packageInfo.packageMetadata.author}</Authors>
+        <PackageIconUrl>https://cloud.google.com/images/gcp-icon-64x64.png</PackageIconUrl>
```

Comment:
Possibly make it clearer that these need editing? We don't want non-Cloud packages with the Cloud icon
(Likewise the repo bits)

Reply:
Done, I think. Please let me know if you had something else in mind.